frontend/modules/recipe/components/Recipe.js

```diff
@@ -185,5 +206,13 @@ var RecipeScheme = injectIntl(React.createClass({
         </div>
       </div>
     );
+  },
+
+  showEditLink: function() {
+    if (this.props.user.id === this.props.data.author) {
```

Comment:
If a user is not logged in, `user` will be null. Trying to access a property of a null value (`id` in this case) breaks bundle. We should check to make sure that user is not null before proceeding. Something like this:

``` javascript
    if (this.props.user !== null) {
      if (this.props.user.id === this.props.data.author) {
        return (
          <Link to={ "/recipe/edit/" + this.props.data.id }>
            <button className="btn btn-primary btn-sm">Edit recipe</button>
          </Link>
        )
      }
    }
```